src/IO/S3/PocoHTTPClient.cpp

```diff
@@ -680,9 +680,7 @@ void PocoHTTPClient::makeRequestInternalImpl(
             addLatency(request, S3LatencyType::Connect, connect_time);
             addLatency(request, first_byte_latency_type, first_byte_time);
         }
-        auto error_message = getCurrentExceptionMessageAndPattern(/* with_stacktrace */ true);
-        error_message.text = fmt::format("Failed to make request to: {}: {}", uri, error_message.text);
-        LOG_INFO(log, error_message);
+        LOG_INFO(log, "Failed to make request to: {}:\n{}", uri, getCurrentExceptionMessage(/* with_stacktrace */ true));
```

Comment:
I prefer avoiding new lines in logs when possible because it makes them harder to grep, but no strong opinion

Reply:
Agree, but there will be newlines anyway, because it prints stacktrace, and in this case it will be better to distinguish it from the message itself.